### PR TITLE
feat: add Polygon zkEVM and zkSync Era

### PR DESCRIPTION
**Pull request by mathiasmoeller**
Add basic information for Polygon zkEVM and zkSync Era

### CODE REVIEW

File: src/chains/supported.chains.ts [hunk 1]
```diff
@@ -866,6 +866,55 @@ export const supportedEVMChains: EVMChain[] = [
...
+    coin: CoinKey.ETH,
+    id: 324,
+    mainnet: true,
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/chains/zksync.png',
+
+    metamask: {
+      chainId: prefixChainId(324),
+      blockExplorerUrls: ['https://zkscan.io/'],
+      chainName: 'ZKSYNC Erа',
```
Comment (maxklenk) [inline src/chains/supported.chains.ts:883]: Why do you use another name than on https://chainlist.org/chain/324? `zkSync Era Mainnet`

File: src/chains/supported.chains.ts [hunk 2]
```diff
@@ -866,6 +866,55 @@ export const supportedEVMChains: EVMChain[] = [
...
+    id: 1101,
+    mainnet: true,
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/chains/zkevm.png',
+    multicallAddress: multicallAddresses[ChainId.PZE],
+
+    metamask: {
+      chainId: prefixChainId(1101),
+      blockExplorerUrls: ['https://zkevm.polygonscan.com'],
+      chainName: 'POLYGONZKEVM',
```
Comment (maxklenk) [inline src/chains/supported.chains.ts:908]: Here as well https://chainlist.org/chain/1101

**Review by maxklenk — COMMENTED** [review]
Looks good. Please just check the chain names passed to the wallets.

File: src/chains/supported.chains.ts (hunk 1 above)
Comment (mathiasmoeller) [reply src/chains/supported.chains.ts:883]: Used the ones from the researched configuration in notion, assumed they were checked alrady. But can change to the chainlist one!

**Review by maxklenk — APPROVED** [review]

**Pull request merged**